src/src.pro

```diff
@@ -4,12 +4,12 @@ PLUGIN_IMPORT_PATH = org/nemomobile/calendar
 TEMPLATE = lib
 CONFIG += qt plugin hide_symbols
 
-QT += qml concurrent
+QT += qml concurrent dbus
```

Comment:
This looks like timed .pc file shortcoming. If using timed requires that, it should be timed's responsibility to require it. Suppose needing Requires: line there.

Reply:
Ah yes, added `BuildRequires:  pkgconfig(Qt5DBus)`

Reply:
That's another thing the timed-devel should handle. If using that needs Qt5DBus, then timed-qt5-devel should have the package dependency.

Reply:
Ok, so I've created sailfishos/timed#6 and removed this BuildRequires.

I've also changed the pro files not to use the pkgconfig file, but instead I've added the `timed-qt5` as a `CONFIG` in the pro files, since timed is providing mkspecs feature files.